src/super_gradients/common/auto_logging/auto_logger.py

```diff
@@ -1,108 +1,113 @@
-import json
-import os
 import logging
-import pkg_resources
+import os
+import sys
+from typing import Union
 
 
 class AutoLoggerConfig:
     """
-    A Class for the Automated Logging Config that is created from the JSON config file (auto_logging_conf)
+    A Class for the Automated Logging Config
     """
 
-    @staticmethod
-    def generate_config_for_module_name(
-        module_name,
-        training_log_path=None,
-        log_level=logging.INFO,
-        max_bytes=10485760,
-        logs_dir_path=None,
-        handlers_list=None,
-    ) -> dict:
+    FILE_LOGGING_LEVEL = os.environ.get("LOG_LEVEL", "DEBUG").upper()
+    CONSOLE_LOGGING_LEVEL = os.environ.get("CONSOLE_LOG_LEVEL", "INFO").upper()
+
+    filename: Union[str, None]
+
+    def __init__(self):
+        self.filename = None
+
+    def _setup_default_logging(self, log_level: str = None) -> None:
         """
-        generate_config_for_module_name - Returns a Config Dict For Logging
-            :param module_name:     The Python Module name to create auto_logging for
-            :param log_level:       Minimal log level to set for the new auto_logging
-            :param max_bytes:       Max size for the log file before rotation starts
-            :param handlers_list:    A list specifying the handlers (Console, etc..) - Better Leave Empty or None
-            :param training_log_path: Path to training log file which all modules of super_gradients will write to. Ignored
-             when set to None.
-            :param logs_dir_path: Path to sg_logs directory (default=None), where module logs will be saved. When set
-                to None- module logs will be saved in ~/sg_logs (created if path does not exist). Main use case is for
-                testing.
-
-
-            :return: python dict() with the new auto_logging for the module
+        Setup default logging configuration. Usually happens when app starts, and we don't have
+        experiment dir yet.
+        The default log directory will be `~/sg_logs`
+        :param log_level: The default log level to use. If None, uses LOG_LEVEL and CONSOLE_LOG_LEVEL environment vars.
+        :return: None
         """
+        self._setup_logging(
+            filename=os.path.expanduser(f"~/sg_logs/last_{os.getppid()}.log"),
```

Comment:
I've added the timestamp alongside with ppid.
There is no _easy_ way to log all events to a single file, when using DDP or DataLoader with num_workers > 1 on Windows platform. 
In both these cases a multiple processes will be spawned and multiple logs may be created.
Therefore the log file will have the parent PID to being able to discriminate the logs corresponding to a single run.

To be resolved in future: How we ensure log rotation in `~/sg_logs` to avoid this directory blow up in size over time? (Best idea so far: Find all logs through regex matching and retain only last N files or at most M bytes)